### DISCUSSION TITLE
how does this work

### DISCUSSION THREAD

**Discussion by Windfave** [Discord-OTP-Forcer/Discord-OTP-Forcer#62] [Community Support]
after setting up the config (email, password, program mode and code mode, token) and turning on start.bat, a chrome tab opens up with discord and logs in, of course the captcha comes in. I complete that and then I am at the enter code screen.
However, nothing seems to be happening neither in the cmd or the chrome tab.

Am I doing something wrong?

**Comment by Inbydev** [comment]
@Windfave could you try with the latest commit?

https://github.com/Derpitron/Discord-OTP-Forcer/archive/refs/heads/main.zip

And follow the tutorial:

https://github.com/Derpitron/Discord-OTP-Forcer/wiki/How-to-setup-and-use

**Comment by Inbydev** [comment]
This probably is an Issue of the past, the program got a lot of updates since then, so re open this discussion if you encounter the same issue with the latest release!

https://codeberg.org/Discord-OTP-Forcer/Discord-OTP-Forcer/releases/tag/v0.5.0-beta

And follow the tutorial:

https://discord-otp-forcer.codeberg.page/en/
